### ISSUE TITLE
ocf:alteeve:server does not follow ocf standards

### ISSUE THREAD

**Issue by fabbione** [ClusterLabs/anvil#392]
[root@an-a01n01 alteeve]# ./server notify
[root@an-a01n01 alteeve]# echo $?
3

https://clusterlabs.org/pacemaker/doc/2.1/Pacemaker_Administration/html/agents.html#actions

notify must not fail, and must exit 0.

**Comment by digimer** [comment]
Changed in my local branch. However, I say in the api or return codes;

```
# 3 - OCF_ERR_UNIMPLEMENTED
#   - The resource agent was instructed to execute an action that we do not implement.
#   - Not all resource agent actions are mandatory. We don't implement 'promote' or 'demote'. We do implement
#     'migrate_to', 'migrate_from', and 'notify'. If we're misconfigured as a master/slave resource, for 
#     example, then will alert the user about this misconfiguration by returning OCF_ERR_UNIMPLEMENTED.
```

We don't implement verify, which is why I used rc:3. Perhaps this should be clarified upstream?

**Comment by fabbione** [comment]
the agent reports metadata with <nofity ..> that means pacemaker assumes that it is implemented. Either remove any reference to notify, or make it return 0.

**Comment by digimer** [comment]
Already changed to 0.

**Comment by digimer** [comment]
Fixed in bf2622a

**Comment by fabbione** [comment]
will not add automation
